src/main/resources/messages.yml

```diff
@@ -33,7 +33,7 @@ Messages:
     message: "&cThere are &a%count%&r&c players missing until the start!"
   is_fleeing_player:
     send_type: "TITLE"
-    message: "&cThe hunters are approaching! Get the Advancement %id%!"
+    message: "&cThe hunters are approaching! Get the Advancement %ah_id%!"
```

Comment:
shouldn't this be `%ah_name%`?

Reply:
yes, my bad

Reply:
btw that depends on your. if you want advancement with namspaced key or just advancement name